core/tests/protractor/editorAndPlayer.js

```diff
@@ -26,11 +26,66 @@ var ExplorationPlayerPage =
 
 describe('Full exploration editor', function() {
   var explorationPlayerPage = null;
+  var parentId2 = null;
```

Comment:
Sorry I missed this before, but these vars shouldn't be here. They should be local to the test. Just put a var before the declarations on lines 49, 57, etc.

Reply:
Ok, done